tools/manifest/vcs.py

```diff
@@ -224,17 +224,61 @@ def dump(self):
         super(MtimeCache, self).dump()
 
 
-class GitIgnoreCache(CacheFile, GitIgnoreCacheType):
-    file_name = "gitignore2.json"
+class GitIgnoreCache(GitIgnoreCacheType):
```

Comment:
When I did this before, I kept this as a subclass of CacheFile, and just overrode dump/load. I guess the alternative option is to introduce a new JSONCacheFile subclass and make dump/load in CacheFile abstract?

Reply:
Or to decide that it was premature abstraction and just not have a base class.